content/_includes/TNHardwareGuide.md

```diff
@@ -160,28 +164,27 @@ Pre-owned or repurposed hardware is popular among TrueNAS users.
 Pay attention to the maximum performance offered by the hot-swap backplanes of a given system. 
 Aim for at least 6 Gbps SATA III support.
 Note that hot-swapping PCIe NVMe devices is not currently supported.
-{{< /tab >}}
-{{< /tabs >}}
+{{< /expand >}}
 
 {{< hint warning >}}
-TrueNAS SCALE does not currently support T10-DIF drives. [Users on our forums have developed a workaround](https://www.truenas.com/community/threads/troubleshooting-disk-format-warnings-in-bluefin.106051/), and we are investigating the issue internally.
+TrueNAS SCALE does not officially support T10-DIF drives. [Users on our forums have developed a workaround for using T10-DIF drives in TrueNAS SCALE](https://www.truenas.com/community/threads/troubleshooting-disk-format-warnings-in-bluefin.106051/), but using unsupported storage devices imposes data-loss risks.
 {{< /hint >}}
 
 ### Storage Device Sizing
 
 [Zpool layout]({{< relref "PoolCreate.md#vdev-layout" >}}) (the organization of LUNs and volumes, in TrueNAS/ZFS parlance) is outside of the scope of this guide. 
-The availability of double-digit terabyte drives raises a question TrueNAS users now have the luxury of asking: How many drives should I use to achieve my desired capacity?
-You can mirror two 16TB drives to achieve 16TB of available capacity, but that does not mean you should.
+The availability of double-digit terabyte drives raises a question TrueNAS users now have the luxury of asking: How many should I use to achieve my desired capacity?
+You can mirror two 16 TB drives to achieve 16 TB of available capacity, but that does not mean you should.
 Mirroring two large drives offers the advantage of redundancy and balancing reads between the two devices, which could lower power draw, but little else.
-The write performance of two large drives, at most, is that of a single drive.
-By contrast, an array of eight 4TB drives offers a wide range of configurations to optimize performance and redundancy at a lower cost.
+The write performance of two large drives is similar to that of a single drive.
+By contrast, an array of eight 4 TB drives offers a wide range of configurations to optimize performance and redundancy at a lower cost.
 If configured as striped mirrors, eight drives could yield four times greater write performance with a similar total capacity.
 You might also consider adding a hot-spare drive with any zpool configuration, which lets the zpool automatically rebuild itself if one of its primary drives fails.
 
 ### Storage Device Burn-In
 
 Spinning disk hard drives have moving parts that are highly sensitive to shock and vibration and wear out with use. 
-Consider pre-flighting every storage device before putting it into production, paying attention to:
+Consider pre-flighting every storage device before putting it into production.
```

Comment:
Not sure removing the "paying attention to" part of this is a good choice as the bullet list below the modified statement appears disjointed.  You could remove the bullet points, or maybe rewrite the sentence to show the relationship to the bullet points below it? 
```suggestion
Consider pre-flighting every storage device before putting it into production, especially:
```